scripts/elasticsearch.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Install Elasticsearch in /tmp and run it.
+set -ev
+mkdir /tmp/elasticsearch
+wget -O - https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.4.tar.gz | tar xz --directory=/tmp/elasticsearch --strip-components=1
```

Comment:
Is it intentional/expected that the URL structure is different for different versions of Elasticsearch?

Reply:
Just the download links the the Es website gives. I did not investigate further!